cmd/activator/request_log_test.go

```diff
@@ -56,48 +56,75 @@ func TestUpdateRequestLogFromConfigMap(t *testing.T) {
 	}
 
 	tests := []struct {
-		name     string
-		url      string
-		body     string
-		template string
-		want     string
+		name string
+		url  string
+		body string
+		data map[string]string
+		want string
 	}{{
-		name:     "empty template",
-		url:      "http://example.com/testpage",
-		body:     "test",
-		template: "",
-		want:     "",
+		name: "empty template",
+		url:  "http://example.com/testpage",
+		body: "test",
+		data: map[string]string{"logging.request-log-template": ""},
```

Comment:
nice, let's touch up for formatting though since the lines get pretty long otherwise (same for all the rows in the table):

```suggestion
		data: map[string]string{
		  "logging.request-log-template": ""
		},
```

Reply:
@julz `go fmt` does this.

Reply:
it will do what I typed above if you add a newline at the right places (go fmt doesn't split long lines automatically, but it's definitely the right thing here and consistent with the rest of the code base, e.g. [1](https://github.com/knative/serving/blob/609a1b8e94a0623463643b11ebc0eaf660677d1b/pkg/webhook/validate_unstructured_test.go#L82), [2](https://github.com/knative/serving/blob/ac91f264dc1826eafa42729b0b968a6e59245cb9/pkg/apis/config/defaults_test.go#L85), [3](https://github.com/knative/serving/blob/ad1aafd077491a876ee45449d770c280e1afc077/pkg/apis/config/features_test.go#L72))

Reply:
just run `go fmt` after adding some newlines as above and it'll do the right thing :)

Reply:
ok let me try.